.travis.yml

```diff
@@ -38,8 +34,8 @@ jobs:
     - stage: dep-cache
       name: "Vendor update"
       script:
-        - source bin/travis_scripts/ensure_glide.sh $BUILD_TOOLS_PATH "v0.13.1"
-        - glide "-home" $GLIDE_HOME install
+        - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
What about `go get ..` way?

Reply:
> It is strongly [recommended](https://github.com/golang/dep#installation) that you use a released version.

And `go get` requires to get sources and build it, so it probably a bit slower than download just binary.